command-line-script.sh

```diff
@@ -38,10 +68,22 @@ monitor.sh &> ${EXECUTION_DIR}/${SUBJECT_FOLDER}/monitor.txt
 cd ${EXECUTION_DIR}                                                                      		  || die ${INITDIR} "Cannot cd to ${EXECUTION_DIR}."
 
 #Adding the reprozip command to trace the processing of subjects
-if [ ${REPROZIP_FLAG} = true ]; then
-  reprozip trace PreFreeSurferPipelineBatch.sh --StudyFolder=$PWD --Subjlist=${SUBJECT_FOLDER} --runlocal || die ${INITDIR} "Pipeline failed."
+if [ ${REPROZIP_FLAG} = true && ${SCRIPT}="-prefs" ]; then
+  reprozip trace PreFreeSurferPipelineBatch.sh --StudyFolder=$PWD --Subjlist=${SUBJECT_FOLDER} --runlocal || die ${INITDIR} "PreFreeSurfer Pipeline failed."
+elif [ ${SCRIPT} = "-fs" && ${REPROZIP_FLAG} = true ];then
+  mv ${LICENSE} ${FREESURFER_HOME}/
+  source ${FREESURFER_HOME}/SetUpFreeSurfer.sh
+  reprozip trace FreeSurferPipelineBatch.sh --StudyFolder=$PWD --Subjlist=${SUBJECT_FOLDER} --runlocal    || die ${INITDIR} "FreeSurfer Pipeline failed."
+elif  [ ${SCRIPT} = "-fs" ];then
+  mv ${LICENSE} ${FREESURFER_HOME}/
+  source ${FREESURFER_HOME}/SetUpFreeSurfer.sh
+  FreeSurferPipelineBatch.sh --StudyFolder=$PWD --Subjlist=${SUBJECT_FOLDER} --runlocal                   || die ${INITDIR} "FreeSurfer Pipeline failed."
+elif [ ${SCRIPT} = "-pfs" && ${REPROZIP_FLAG} = true ];then
+  reprozip trace PostFreeSurferPipelineBatch.sh --StudyFolder=$PWD --Subjlist=${SUBJECT_FOLDER} --runlocal|| die ${INITDIR} "PostFreeSurfer Pipeline failed."
+elif [ ${SCRIPT} = "-pfs" ];then
+  PostFreeSurferPipelineBatch.sh --StudyFolder=$PWD --Subjlist=${SUBJECT_FOLDER} --runlocal               || die ${INITDIR} "PostFreeSurfer Pipeline failed."
 else
-  PreFreeSurferPipelineBatch.sh --StudyFolder=$PWD --Subjlist=${SUBJECT_FOLDER} --runlocal                || die ${INITDIR} "Pipeline failed."
+  PreFreeSurferPipelineBatch.sh --StudyFolder=$PWD --Subjlist=${SUBJECT_FOLDER} --runlocal                || die ${INITDIR} "PreFreeSurfer Pipeline failed."
```

Comment:
I think you need another command-line-script.sh for Freesurfer. You should eventually have two: one for PreFreesurfer and one for Freesurfer. You may want to restructure a bit the repo to separate the files used in each of these containers.